lib/cocoapods/installer/xcode/pods_project_generator/target_installation_result.rb

```diff
@@ -33,21 +33,28 @@ class TargetInstallationResult
           #
           attr_reader :test_resource_bundle_targets
 
+          # @return [Array<PBXNativeTarget>] test_app_host_targets
+          #         The test app host native targets that were produced for this target. Can be empty.
+          #
+          attr_reader :test_app_host_targets
```

Comment:
this seems to be unused?

Reply:
it is but its correct. This is the entire result of `pod_target_installer.rb` it might be useful to carry through this information. I know...YAGNI. but whatevez